addons/payment_custom/views/payment_provider_views.xml

```diff
@@ -14,6 +24,19 @@
                     {'invisible': [('code', '=', 'custom')]}
                 </attribute>
             </group>
+            <field name="pending_msg" position="after">
+                <div class="o_row replace_pending_msg" colspan="2"
+                     attrs="{'invisible': ['|', ('code', '!=', 'custom'), ('custom_mode', '!=', 'wire_transfer')]}">
```

Comment:
Out of curiosity, putting this attrs in the button and getting rid of the div... Does it change a lot in the view?